ProjectCoimbra.UWP/Project.Coimbra.sln

```diff
@@ -191,6 +193,36 @@ Global
 		{E32ED1E8-D9AC-4215-B0EC-B9B038DEAE1F}.Release|x64.Build.0 = Release|Any CPU
 		{E32ED1E8-D9AC-4215-B0EC-B9B038DEAE1F}.Release|x86.ActiveCfg = Release|Any CPU
 		{E32ED1E8-D9AC-4215-B0EC-B9B038DEAE1F}.Release|x86.Build.0 = Release|Any CPU
+		{060B8571-854F-4C66-B802-FC93211FE12B}.Coverage|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Please remove these new platforms from your .csproj and then remove them from here too. They're going to result in platforms we can't build, which will make things more confusing

Reply:
Spoke with Muiris offline and this will be addressed in a separate PR.